packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -1,6 +1,9 @@
 export interface CrudGeneratorOptions {
     targetDirectory: string;
-    requiredPermission?: string[] | string;
+    requiredPermission?:
+        | string[]
+        | string
+        | { list: string | string[]; create: string | string[]; update: string | string[]; delete: string | string[] };
```

Comment:
it would be nice if this wasn't string but permission-string, as defined in @RequiredPermission-decorator.

Reply:
Why isn't this possible?

Reply:
I think it could be possible, I didn't tried it by now.